node/runtime/src/lib.rs

```diff
@@ -747,7 +747,11 @@ impl Runtime {
 
     fn print_log(log: &[LogEntry]) {
```

Comment:
why not instead check inside print_log

if log.is_empty { return; }

then users of this function won't need to add this guard every time.

Reply:
Good point

Reply:
Fixed